src/modules/fancyzones/lib/util.cpp

```diff
@@ -5,6 +5,7 @@
 #include <common/common.h>
 #include <common/dpi_aware.h>
 
+#include <array>
```

Comment:
Why is this required? No other changes to this file have been made.

Reply:
std::array is used few times in that file. Just to avoid transitive include